### PR TITLE
Add Darwin 22 SCA files to specs

### LINKED ISSUE #2137: Add SPECs for macOS Ventura
|Wazuh version|Install type|Action performed|Platform|
|---|---|---|---|
| 4.4.2 | Agent | Install | macOS Ventura|

Hello team, we need SPECS for this new SCA file:
PR - https://github.com/wazuh/wazuh/pull/16027
Branch - https://github.com/wazuh/wazuh/tree/macOS-Ventura-SCA

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|Closes #2137 |

## Description

This PR adds the necessary changes to include the new Darwin 22 SCA files to de SPECS.

## Tests

- Build the package in any supported platform
  - [x] Linux
  - [ ] Windows
  - [ ] macOS
  - [ ] Solaris
  - [ ] AIX
  - [ ] HP-UX
- [x] Package installation
- [x] Package upgrade
- [ ] Package downgrade
- [ ] Package remove
- [ ] Package install/remove/install
- [ ] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [x] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] Package install/remove/install
  - [ ] Package install/purge/install
  - [ ] Check file permissions after installing the package
- Tests for macOS
  - [ ] Test the package from macOS Sierra to Mojave
- Tests for Solaris
  - [ ] Test the package on Solaris 10
  - [ ] Test the package on Solaris 11
  - [ ] Check file permissions on Solaris 11 template
- Tests for IBM AIX
  - [ ] `%files` section is correctly updated if necessary
  - [ ] Check the changes from IBM AIX 5 to 7

### CODE REVIEW

**Review by verdx — APPROVED** [review]

**Review by 72nomada — APPROVED** [review]

File: macos/package_files/build.sh [hunk 1]
```diff
@@ -72,6 +72,7 @@ function build() {
     cp ${SOURCES_PATH}/etc/templates/config/darwin/19/sca.files ${INSTALLATION_SCRIPTS_DIR}/sca/darwin/19/
     cp ${SOURCES_PATH}/etc/templates/config/darwin/20/sca.files ${INSTALLATION_SCRIPTS_DIR}/sca/darwin/20/
     cp ${SOURCES_PATH}/etc/templates/config/darwin/21/sca.files ${INSTALLATION_SCRIPTS_DIR}/sca/darwin/21/
+    cp ${SOURCES_PATH}/etc/templates/config/darwin/21/sca.files ${INSTALLATION_SCRIPTS_DIR}/sca/darwin/22/
```
Comment (jnasselle) [inline macos/package_files/build.sh:75]:
```suggestion
    cp ${SOURCES_PATH}/etc/templates/config/darwin/22/sca.files ${INSTALLATION_SCRIPTS_DIR}/sca/darwin/22/
```

File: debs/SPECS/wazuh-manager/debian/rules [hunk 2]
```diff
@@ -166,6 +167,7 @@ override_dh_install:
 	cp etc/templates/config/darwin/18/sca.files ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/darwin/19
 	cp etc/templates/config/darwin/20/sca.files ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/darwin/20
 	cp etc/templates/config/darwin/21/sca.files ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/darwin/21
+	cp etc/templates/config/darwin/21/sca.files ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/darwin/22
```
Comment (jnasselle) [inline debs/SPECS/wazuh-manager/debian/rules:170]:
```suggestion
	cp etc/templates/config/darwin/22/sca.files ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/darwin/22
```

Comment (jnasselle) [reply debs/SPECS/wazuh-manager/debian/rules:170]: https://github.com/wazuh/wazuh/blob/macOS-Ventura-SCA/etc/templates/config/darwin/22/sca.files

File: macos/package_files/build.sh (hunk 1 above)
Comment (jnasselle) [reply macos/package_files/build.sh:75]: https://github.com/wazuh/wazuh/blob/macOS-Ventura-SCA/etc/templates/config/darwin/22/sca.files

**Review by jnasselle — CHANGES_REQUESTED** [review]

**Review by davidcr01 — APPROVED** [review]

**Review by verdx — APPROVED** [review]

**Review by rauldpm — APPROVED** [review]

**Comment by c-bordon** [comment]
## Tested in Jenkins:
Builds: https://ci.wazuh.info/view/Packages/job/Packages_builder_tier/2926
Upgrade tests: https://ci.wazuh.info/view/Tests/job/Test_upgrade_tier/2060

**Review by jnasselle — APPROVED** [review]

**Pull request merged**